paimon-core/src/main/java/org/apache/paimon/catalog/CachingCatalog.java

```diff
@@ -227,27 +224,13 @@ public Table getTable(Identifier identifier) throws TableNotExistException {
             return table;
         }
 
-        if (isSpecifiedSystemTable(identifier)) {
-            Identifier originIdentifier =
-                    new Identifier(
-                            identifier.getDatabaseName(),
-                            identifier.getTableName(),
-                            identifier.getBranchName(),
-                            null);
+        if (identifier.isSystemTable()) {
+            Identifier originIdentifier = identifier.toOriginTable();
             Table originTable = tableCache.getIfPresent(originIdentifier);
             if (originTable == null) {
                 originTable = wrapped.getTable(originIdentifier);
                 putTableCache(originIdentifier, originTable);
             }
-            table =
-                    SystemTableLoader.load(
-                            Preconditions.checkNotNull(identifier.getSystemTableName()),
-                            (FileStoreTable) originTable);
-            if (table == null) {
-                throw new TableNotExistException(identifier);
-            }
-            putTableCache(identifier, table);
-            return table;
```

Comment:
why removing this?

Reply:
the line 253~254 do the same thing, and this can cover by testInvalidateSysTablesIfBaseTableIsDropped

Reply:
Oh, I see, here we want to reuse the table to wrap the system table